marathon-bigip-ctlr.py

```diff
@@ -577,10 +577,20 @@ def get_apps(apps, health_check):
                 marathon_app.app['labels']['F5_PARTITION']
         marathon_apps.append(marathon_app)
 
-        service_ports = app['ports']
-        logger.debug("Application service ports = %s", (repr(service_ports)))
-        logger.debug("Labels for app %s: %s", app['id'],
-                     marathon_app.app['labels'])
+
+        del app['ports']
```

Comment:
I don't think this should be here?

Reply:
Yes, I just closed this -- that's my test code.